cygnus-common/src/main/java/com/telefonica/iot/cygnus/sinks/Enums.java

```diff
@@ -14,6 +14,6 @@
     /**
      * Available data models for all the sinks.
      */
-    public enum DataModel { DMBYSERVICE, DMBYSERVICEPATH, DMBYENTITY, DMBYENTITYTYPE, DMBYATTRIBUTE }
+    public enum DataModel { DMBYSERVICE, DMBYSERVICEPATH, DMBYENTITY, DMBYENTITYTYPE, DMBYATTRIBUTE, DMBYENTITYID }
```

Comment:
Maybe is a naïve question :) but... where is the code that translate the string "dm-by-entity-id" to DMBYENTITYID ? Is some kind of generic uppercase + dash removal function?

Reply:
In the configure method , in NGSISynk.java: 
`@Override
    public void configure(Context context) {
        String dataModelStr = context.getString("data_model", "dm-by-entity");`

Reply:
[NGSISink read it on the agent properties](https://github.com/telefonicaid/fiware-cygnus/blob/master/cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSISink.java#L209) and then CKAN sinks inherits it